### PR TITLE
add basepath along with filepath into yaml comment

### PR DESCRIPTION
**Pull request by itscaro**

### CODE REVIEW

**Comment by yxxhero** [comment]
@itscaro do ready for review?

**Comment by yxxhero** [comment]
@itscaro why output the fullpath?

**Comment by yxxhero** [comment]
@itscaro In my mind, helm don't ouput the fullpath.

**Comment by itscaro** [comment]
This is just for the comment, the result of helmfile build contain `# Source: helmfile.yaml`. In my case I have many helmfile.yaml in multiple directories, the actual comment is not helpful to identify the original helmfile in the merged one.

File: pkg/state/state.go [hunk 1]
```diff
@@ -3356,3 +3356,7 @@ func (st *HelmState) getOCIChart(release *ReleaseSpec, tempDir string, helm helm
 
 	return &chartPath, nil
 }
+
+func (st *HelmState) FullFilePath() string {
```
Comment (yxxhero) [inline pkg/state/state.go:3360]: @itscaro add some test will be better?

Comment (itscaro) [reply pkg/state/state.go:3360]: @yxxhero, is it OK like this?

Comment (yxxhero) [reply pkg/state/state.go:3360]: good. so great. @itscaro

**Review by yxxhero — APPROVED** [review]

**Pull request merged**